### ISSUE TITLE
rnn_transducer test error

### ISSUE THREAD

**Issue by christina284** [TensorSpeech/TensorFlowASR#269]
Hello, I am trying to run test.py form the rnn_transducer example but the following error keeps coming up... 
![Screenshot from 2022-09-13 03-22-21](https://user-images.githubusercontent.com/30560364/189781008-f5848b91-a0af-4bbb-8e1a-c2f43d3ceab1.png)
Is this an error or am I doing something wrong?

My config.yml file looks like this:
`
speech_config:
  sample_rate: 16000
  frame_ms: 25
  stride_ms: 10
  num_feature_bins: 80
  feature_type: log_mel_spectrogram
  preemphasis: 0.97
  normalize_signal: True
  normalize_feature: True
  normalize_per_frame: True
  center: False
  top_db: null

decoder_config:
  vocabulary: ./vocabularies/librispeech/dev-other.subwords
  target_vocab_size: 1024
  max_subword_length: 4
  blank_at_zero: True
  beam_width: 5
  norm_score: True

model_config:
  name: rnn_transducer
  encoder_reductions:
    0: 3
    1: 2
  encoder_dmodel: 320
  encoder_rnn_type: lstm
  encoder_rnn_units: 1024
  encoder_nlayers: 8
  encoder_layer_norm: True
  prediction_embed_dim: 320
  prediction_embed_dropout: 0.0
  prediction_num_rnns: 2
  prediction_rnn_units: 1024
  prediction_rnn_type: lstm
  prediction_projection_units: 320
  prediction_layer_norm: True
  joint_dim: 320
  joint_activation: tanh

learning_config:
  train_dataset_config:
    use_tf: True
    augmentation_config:
      feature_augment:
        time_masking:
          num_masks: 10
          mask_factor: 100
          p_upperbound: 0.05
        freq_masking:
          num_masks: 1
          mask_factor: 27
    data_paths:
      - LibriSpeech/train-clean-100/transcripts.tsv
    tfrecords_dir: null
    shuffle: True
    cache: True
    buffer_size: 100
    drop_remainder: True
    stage: train

  eval_dataset_config:
    use_tf: True
    data_paths:
      - LibriSpeech/dev-clean/transcripts.tsv
    tfrecords_dir: null
    shuffle: False
    cache: True
    buffer_size: 100
    drop_remainder: True
    stage: eval

  test_dataset_config:
    use_tf: True
    data_paths:
      - LibriSpeech/test-other/transcripts.tsv
    tfrecords_dir: null
    shuffle: False
    cache: True
    buffer_size: 100
    drop_remainder: True
    stage: test

  optimizer_config:
    class_name: adam
    config:
      learning_rate: 0.0001

  running_config:
    batch_size: 2
    num_epochs: 20
    checkpoint:
      filepath: examples/rnn_transducer/checkpoints/{epoch:02d}.h5
      save_best_only: False
      save_weights_only: True
      save_freq: epoch
    states_dir: examples/rnn_transducer/states
    tensorboard:
      log_dir: examples/rnn_transducer/tensorboard
      histogram_freq: 1
      write_graph: True
      write_images: True
      update_freq: epoch
      profile_batch: 2
`

**Comment by usimarit** [comment]
@christina284 Please make sure you passed the flag `--subwords` to the `python test.py --subwords`
